frb_codegen/src/library/codegen/parser/type_parser/enum_or_struct.rs

```diff
@@ -109,10 +109,11 @@ fn compute_name_and_wrapper_name(
     mirror: bool,
 ) -> (NamespacedName, Option<String>) {
     let name = ident.to_string();
+    let namespace = NamespacedName::new(namespace.clone(), name);
```

Comment:
nit: this variable may be named `namespaced_name` (instead of `namespace` which means a namespace without a name)